README.md

```diff
@@ -23,18 +28,21 @@ terra
 1. Java 11
 2. Docker 20.10.2 (Must be running)
 
-#### Authentication
+#### Login
 1. Use a Google account that is not a Google/Verily corporate account.
 2. `terra auth login` launches an OAuth flow that pops out a browser window with a warning login
 page ("! Google hasn't verified this app"). This shows up because the CLI is not yet a Google-verified
 app. Click through the warnings ("Advanced" -> "Go to ... (unsafe)") to complete the login.
 
-Note: The default server has been temporarily changed from `terra-dev` to `wchamber-dev`.
-This is because the Terra dev services are behind a firewall that requires users to be
-on the Broad VPN. There is an open ticket with Broad DevOps to address this. In the meantime,
-personal development deployments are apparently not behind the firewall, so we're using
-one of those as a temporary workaround. If you do have access to the Broad VPN, you can run 
-`terra server set terra-dev` to change it back to official Terra dev environment.
+#### Spend Profile Access
+In order to spend money (e.g. by creating a project and resources within it) in Terra, you need
+access to a billing account via a spend profile. Currently, there is a single spend profile used
+by Workspace Manager. Your email needs to either be added as a user of that spend profile or added
+to a Terra group that is a user of that spend profile. This needs to be done by someone else with
+owner access to that spend profile.
+
+- [Preferred] Add a user to a Terra group that is a user of the spend profile with `terra groups add-user --group=enterprise-pilot-testers --policy=admin mmdevverily4@gmail.com`
```

Comment:
Does --policy=admin mean the added user can add other users to the group. Do we want to default to giving every user the permission to add other users?

Reply:
Yes. I think for the solutions team, we want to just give them admin so they don't need to ask us every time they want to use a new account. But for future users, probably not. Updated to default to member, but with instructions for admin also.